Dockerfile.art

```diff
@@ -36,4 +45,10 @@ USER 1001
 
 COPY --from=builder /usr/src/app/dist /usr/share/nginx/html
```

Comment:
This needs to be changed to use `web-builder` instead as `builder` no longer exists

I agree with duplicating the dist and putting it in both the `/usr/share/nginx/html` and at the `/opt/app-root/web`. It should make it a lot easier to transition to just the go binary

Reply:
Thnx for the suggestion, indeed is the `web-builder` now, is fixed now